extensions/src/AWSSDK.Extensions.Bedrock.MEAI/AWSSDK.Extensions.Bedrock.MEAI.nuspec

```diff
@@ -15,17 +15,17 @@
       <group targetFramework="net472">
         <dependency id="AWSSDK.Core" version="4.0.0.2" />
         <dependency id="AWSSDK.BedrockRuntime" version="4.0.0.1" />
-        <dependency id="Microsoft.Extensions.AI.Abstractions" version="9.4.3-preview.1.25230.7" />
+        <dependency id="Microsoft.Extensions.AI.Abstractions" Version="9.5.0-preview.1.25262.9" />
```

Comment:
Changing the upper case `V` for `version` broke the packaging, Can you reset back to lowercase `version`.
```
Attempting to build package from 'AWSSDK.Extensions.Bedrock.MEAI.nuspec'.
4962 | The 'Version' attribute is not declared.
4963 | C:\codebuild\tmp\output\src102953425\src\aws-sdk-net\buildtools\build.proj(509,9): error MSB3073: The command "..\..\..\sdk\.nuget\NuGet.exe pack ./AWSSDK.Extensions.Bedrock.MEAI.nuspec -BasePath . -OutputDirectory ../../../Deployment/nuget" exited with code 1.
```